server/public/model/config.go

```diff
@@ -2908,10 +2912,18 @@ func (s *DataRetentionSettings) SetDefaults() {
 		s.MessageRetentionDays = NewInt(DataRetentionSettingsDefaultMessageRetentionDays)
 	}
 
+	if s.MessageRetentionHours == nil {
+		s.MessageRetentionHours = NewInt(DataRetentionSettingsDefaultMessageRetentionHours)
+	}
```

Comment:
Should we keep this `nil` instead? For new installations, we won't use that setting any longer.

Reply:
Did you mean to do that for `days`? I already set the default for `days` to `0`